src/CMakeLists.txt

```diff
@@ -10,7 +10,21 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR})
 
 add_subdirectory(crypto)
 add_subdirectory(univalue)
+
+add_custom_target(generate_build_info
+  BYPRODUCTS ${PROJECT_BINARY_DIR}/src/obj/build.h
```

Comment:
Note that the `obj` is catering to autotools, trying to keep the generated files away from the source ones. In a follow-up after CMake merge, we could drop the `obj/` prefix. Mind adding a note?

Reply:
Thanks! [Done](https://github.com/hebasto/bitcoin/pull/51#issuecomment-1815013786).